### PR TITLE
[stable/openldap] Add deployment strategy configuration

### PR DESCRIPTION
**Pull request by kruc**
Signed-off-by: Jacek Ciwiś <jacek.ciwis@thesoftwarehouse.pl>

#### Checklist
- [x] [DCO](https://www.helm.sh/blog/helm-dco/index.html) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md

### CODE REVIEW

File: stable/openldap/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 name: openldap
 home: https://www.openldap.org
-version: 0.2.4
+version: 0.2.5
```
Comment (cpanato) [inline stable/openldap/Chart.yaml]: can you please bump to 0.3.0 since this is not a bug fix?

**Comment by cpanato** [comment]
@kruc did you test an upgrade chart from an existing to this?

File: stable/openldap/Chart.yaml (hunk 1 above)
Comment (kruc) [reply stable/openldap/Chart.yaml]: yes, sure, thanks for that

**Comment by kruc** [comment]
@cpanato I tested it now and I have following error
`Error: UPGRADE FAILED: Deployment.apps "openldap" is invalid: spec.strategy.rollingUpdate: Forbidden: may not be specified when strategy `type` is 'Recreate'`.
Thank you for paying attention, fix in progress

**Review by cpanato — APPROVED** [review]

**Comment by benlangfeld** [comment]
@kruc Surely this didn't actually work for you? The `rollingUpdate: null` key is only actually set in the template output if you remove the `strategy: {}` from the chart's `values.yaml`.

**Pull request merged**